module/research/research.py

```diff
@@ -341,11 +341,12 @@ def handle_research_reward(self):
         if not self.config.ENABLE_RESEARCH_REWARD:
             return False
 
-        if not self.appear(RESEARCH_FINISHED) and not self.appear(RESEARCH_PENDING, offset=(20, 20)):
-            logger.info('No research finished or pending')
+        if not self.appear(RESEARCH_PENDING, offset=(20, 20)):
+            logger.info('No research pending')
```

Comment:
When drop screenshot enabled, research receive is in page_research, because research stats need research name.
So here needs RESEARCH_FINISHED.